src/components/dialogs/voltage-level-creation-dialog.js

```diff
@@ -139,7 +139,7 @@ const BusBarConnexion = ({
             defaultValue && defaultValue.fromBBS
                 ? getBusbarSectionById(fieldProps, defaultValue?.fromBBS)
                 : '',
-        values: fieldProps,
+        values: fieldProps.filter((val) => val?.id !== ''),
```

Comment:
'?.' could return undefined, so filtering against empty string may be too precise, no ?